javascript/modules/client/README.md

```diff
@@ -441,23 +500,241 @@ console.log(proposals);
+const actions: DaoAction[] = [
+  // See the "Action encoders" section below
+  { to: "0x1234...", value: BigInt(100), data: new Uint8Array([1, 2, 3, 4]) },
+];
+const proposalCreationParams: ICreateProposalParams = {
+  metadataUri: "<uri>", // Following the EIP-4824
+  actions,
+  // TODO: Clarify => block number or timestamp?
+  startDate: 1234,
+  endDate: 2345,
+  executeIfPassed: true,
+  creatorVote: VoteOption.YES,
+};
 
-### Voting on a multisig proposal
+const estimatedGas = await client.estimation.createProposal(
+  proposalCreationParams,
+);
+console.log(estimatedGas.average); // bigint
+console.log(estimatedGas.max); // bigint
 
-- **TODO**
+for await (
+  const step of client.methods.createProposal(
+    proposalCreationParams,
+  )
+) {
+  switch (step.idx) {
+    case DaoDepositSteps.CREATING:
+      console.log(step.txHash); // 0xb1c14a49...
+      break;
+    case DaoDepositSteps.DONE:
+      console.log(step.proposalId); // 0x1234...
+      break;
+  }
+}
 
-### Loading the list of members (multisig)
 
-- **TODO**
+```
+
+### Voting on a Multisig proposal
+```ts
+  // const contextParams: ContextPluginParams = {...} 
+  const context = new ContextPlugin(contextParams)
+  const client = new ClientMultisig(context)
+  const estimatedGas = await client.estimation.voteProposal(
+    proposalId, VoteOptions.YES
+  );
+  console.log(estimatedGas.average); // bigint
+  console.log(estimatedGas.max); // bigint
+
+  for await (const step of client.methods.voteProposal(proposalId, VoteOptions.YES)) {
+    switch (step.key) {
+      case VoteProposalStep.VOTING:
+        expect(typeof step.txHash).toBe("string");
+        expect(step.txHash).toMatch(/^0x[A-Fa-f0-9]{64}$/i);
+        break;
+      case VoteProposalStep.DONE:
+        expect(typeof step.voteId).toBe("string");
+        expect(step.voteId).toMatch(/^0x[A-Fa-f0-9]{64}$/i);
+        break;
+      default:
+        throw new Error(
+          "Unexpected vote proposal step: " + Object.keys(step).join(", "),
+        );
+    }
+  }
+
+```
+
+### Loading the list of members (Multisig)
+```ts
+  const context = new ContextPlugin(contextParams)
+  const client = new ClientMultisig(context)
+  const members = await client.methods.getMemebers()
+  /*
+    [
+      "0x1234567890123456789012345678901234567890",
+      "0x2345678901234567890123456789012345678901",
+      "0x3456789012345678901234567890123456789012",
+      "0x4567890123456789012345678901234567890123",
+    ]
+  */
+
+```
+
+### Loading the a proposal by proposalId (Multisig)
+```ts
+  const context = new ContextPlugin(contextParams)
+  const client = new ClientMultisig(context)
+  const proposals = await client.methods.getProposal(
+    "0x1234567890123456789012345678901234567890_0x0"
+  )
+  /*
+    {
+      id: "0x1234567890123456789012345678901234567890_0x0",
+      daoAddress: "0x56fb7bd9491ff76f2eda54724c84c8b87a5a5fd7",
+      daoName: "DAO 1",
+      creator: "0x8367dc645e31321CeF3EeD91a10a5b7077e21f70",
```

Comment:
```suggestion
      creatorAddress: "0x8367dc645e31321CeF3EeD91a10a5b7077e21f70",
```

Reply:
This naming was specified by the app team, to be `creator`